tests/homeTasks/homeTask16.spec.js

```diff
@@ -0,0 +1,436 @@
+import { expect, test } from "@playwright/test";
+
+test.describe("New user registration", () => {
+  let page;
+
+  test.beforeAll(async ({ browser }) => {
+    const ctx = await browser.newContext();
+    page = await ctx.newPage();
+    await page.goto("/");
+  });
+
+  test("Open login form", async () => {
+    const loginButton = page.locator("button", {
+      hasText: "Sign in",
+    });
+
+    await loginButton.click("log in form should opens");
```

Comment:
move it to beforeEach hook

Reply:
go to page and open popup before each tests. This will make your tests independent